packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1070,6 +1087,15 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             this.deltaManager.on("op", this.onOp);
         }
 
+        this.logger.sendTelemetryEvent({
+            eventName: "ContainerLoadStats",
+            containerLoadDataStoreCount: this.dataStores.containerLoadDataStoreCount,
+            referencedDataStoreCount: this.dataStores.referencedDataStoreCount,
+            createContainerRuntimeVersion: this.createContainerMetadata.createContainerRuntimeVersion,
```

Comment:
Any reason not to do ...this.createContainerMetadata here? I.e. include all current and future properties?
If we do not want to log them all, maybe createContainerMetadata  should be restructured to have two groups of properties (i.e. two properties, each an object with properties), one that we log, and one that we do not (assuming it's possible without breaking back compat)?

Reply:
Actually, why not dump more info, i.e. ...this.formMetadata()?

Reply:
at first I thought `...this.createContainerMetadata` doesn't work and it has to be individual properties...

Reply:
do we want to log all of metadata? including message?

Reply:
I do not see why not :)